README.md

```diff
@@ -584,7 +584,7 @@ instrumentation.opentelemetry.io/inject-sdk: "true"
 
 #### Controlling Instrumentation Capabilities
 
-The operator allows specifying, via the flags, which languages the Instrumentation resource may instrument.
+The operator allows specifying, via the flags (`--set manager.extraArgs={"<Name-of-the-gate>=<Value>"}`), which languages the Instrumentation resource may instrument.
```

Comment:
On the operator it self it would look a bit different.
https://github.com/open-telemetry/opentelemetry-operator/blob/d310e0967f6623c2af0c0926120787a140697852/.github/workflows/e2e.yaml#L51